tailwind.config.js

```diff
@@ -164,6 +164,13 @@ module.exports = {
       content: {
         retro: 'attr(data-text)',
       },
+      translate: {
+        'negative-0.5': '-0.25rem',
```

Comment:
Can't we use `-0.5` for this? Usually works with other tailwind rules (check `inset` or `margin`)

Reply:
Sure! Thank you, didn't notice that. Fixed.